src/script/components/Modals/PrimaryModal/PrimaryModalTypes.ts

```diff
@@ -79,6 +79,7 @@ export interface ModalContent {
   currentType: string | PrimaryModalType;
   inputPlaceholder: string;
   message: React.ReactNode;
+  messageHtml?: string;
```

Comment:
why do we have both now? what is the diff between `htmlMessage` and `messageHtml`?

Reply:
we had both before as well. depends through which flow the modal is initiated.

Reply:
let's just use `htmlMessage` from now on (and deprecate it)